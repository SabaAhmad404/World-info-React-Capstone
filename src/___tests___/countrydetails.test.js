import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import { Provider } from 'react-redux';
import store from '../redux/configstore';
import CountryDetails from '../components/countrydetails';

configure({ adapter: new Adapter() });

describe('Details of country', () => {
  it('renders correctly detaails', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <CountryDetails />
      </Provider>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
