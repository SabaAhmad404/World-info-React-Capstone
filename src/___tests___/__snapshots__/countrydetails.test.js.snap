// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details of country renders correctly detaails 1`] = `ShallowWrapper {}`;
